SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKSimulatedPlayerDeployment.cpp

```diff
@@ -564,6 +566,20 @@ FReply SSpatialGDKSimulatedPlayerDeployment::OnLaunchClicked()
 		return FReply::Handled();
 	}
 
+	if (SpatialGDKSettings->IsSimulatedPlayersEnabled())
+	{
+		IPlatformFile& PlatformFile = FPlatformFileManager::Get().GetPlatformFile();
+		FString BuiltWorkerFolder = GetDefault<USpatialGDKEditorSettings>()->GetBuiltWorkerFolder();
+		FString BuiltSimPlayersName = TEXT("UnrealSimulatedPlayer@Linux.zip");
```

Comment:
Slightly worried whether we have this name used anywhere else? In the docs?

Reply:
And that we should have this stored somewhere globally accessible and only have one place that sets it?

What is the concern with docs?

Reply:
Yes, to the first part.
I meant, how will people know that to build simplayers, they need to have this exact assembly name?

Reply:
Testing building SimPlayers manually it seems UnrealSimulatedPlayer@Linux is alwas the ouput of BuildWorker.bat. In otherwords it doesnt matter what the customer will put in as a name for the SimPlayer build. We do have docs on this:

![image](https://user-images.githubusercontent.com/56311103/75368928-692dec00-58ba-11ea-9c64-4c829979ddd3.png)

Reply:
As long as they follow the docs they output zipped file should be "UnrealSimulatedPlayer@Linux.zip"

Reply:
Looking in the code base I can not find another use of UnrealSimulatedPlayer@Linux

Reply:
Ah, okay, <YourProject>SimulatedPlayer dictates the name of the assembly and it's in docs, all good.